packages/react-select/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-select",
-  "version": "4.3.1",
+  "version": "5.0.0-next.0",
```

Comment:
Should this be `5.0.0-beta.1`?

Reply:
Yep, I've changed it from `next` to `beta` but we'll start at `.0` because that's a change sets thing we can't change at the moment